packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.ts

```diff
@@ -294,26 +294,32 @@ export function makeMongoDbEmbeddedContentStore({
 }
 
 type MongoDbAtlasVectorSearchFilter = {
-  sourceName?: string;
-  "metadata.version.label"?: string;
+  sourceName?: string | { $in: string[] };
```

Comment:
just confirming, are these filters already on the database? and should there be tests for this?

Reply:
just looked at the clusters, and it seems the answer is yes. closing thread.